README.md

```diff
@@ -22,8 +22,7 @@ FairScale was designed with the following values in mind:
 
 ## What's New:
 
-* September 2021 [`master` branch renamed to `main`](https://github.com/github/renaming).
-* September 2021 [fairscale 0.4.1 was released](https://github.com/facebookresearch/fairscale/releases/tag/v0.4.1).
+* November 2021 [fairscale 0.4.2 was released](https://github.com/facebookresearch/fairscale/releases/tag/v0.4.2).
```

Comment:
Can we maintain the last 5 news items? Having a single one looks odd.

Reply:
@anj-s I'll retain the Sep 2021 items. please provide two more items.

Reply:
1. We have a new experimental API called the [LayerwiseMemoryTracker](https://fairscale.readthedocs.io/en/latest/tutorials/layer_memory_tracking.html) to help track, visualize and suggest fix for memory issues occurring during the forward/backward pass of your models. 
2. Introducing [SlowMoDistributedDataParallel](https://fairscale.readthedocs.io/en/latest/tutorials/slowmo_ddp.html) API, a distributed training wrapper that is useful on clusters with slow network interconnects (eg Ethernet).

Reply:
thanks, added to the readme.